sensor_msgs/Image frame
string detections